packages/esm-patient-list-app/src/api/queries.ts

```diff
@@ -113,3 +117,40 @@ export function useRemovePatientsFromOfflinePatientListMutation() {
     }
   });
 }
+
+export function usePatientListDetails(patientListUuid: string): [OpenmrsCohort, KeyedMutator<{ data: OpenmrsCohort }>] {
+  if (patientListUuid) {
+    const {
+      data: cohortDetails,
+      error,
+      mutate,
+    } = useSWR<{ data: OpenmrsCohort }, Error>(`${cohortUrl}/cohort/${patientListUuid}`, openmrsFetch);
+    if (error) {
+      throw error;
+    }
+    return [cohortDetails?.data, mutate];
+  }
+}
+
+interface CohortResponse<T = any> {
```

Comment:
This type is already exported in another file (`esm-patient-list-app/src/api/types.ts`). The duplication here can be avoided.